avocado/utils/kernel.py

```diff
@@ -22,7 +22,7 @@
 import os
 import shutil
 import tempfile
-from distutils.version import LooseVersion  # pylint: disable=E0611
+from packaging import version as versobjc
```

Comment:
Your PR it is talking about time.monotonic() and here we have changes about distutils. Maybe you need a rebase or fork from master to avoid having this changes here.

Reply:
Oh my bad. The changed file from another PR slipped in here. I'll restore the original kernel.py right away.

Reply:
(I think I would wait on #4253 being taken in correctly. Once, it happens I would know just what all I should do and what all I should avoid.)

Reply:
Or you could rebase this PR based on master, to avoid dependencies, since this PR is not related to #4253.